wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -888,24 +888,29 @@ public final class SessionManager: NSObject, SessionManagerType {
         }
     }
 
-    private func switchAnalyticsUser() {
+    private func switchAnalyticsUser(for userSession: ZMUserSession) {
```

Comment:
I think this should be `to` instead of `for`. Small thing but I think otherwise it could be confusing.